Hi @sma-github thanks for this PR.
It failed on Linux platform due to this:
```
CMake Error at scripts/cmake/vcpkg_find_acquire_program.cmake:295 (message):
  Could not find doxygen.  Please install it via your package manager.
Call Stack (most recent call first):
  ports/ftgl/portfile.cmake:1 (vcpkg_find_acquire_program)
  scripts/ports.cmake:94 (include)
```
Since `doxygen `is only for windows now. So we need to install it manually on other platform.
I test it on my local. Even though I have installed `doxygen` , it still build failed.
```
/usr/include/c++/5/bits/c++0x_warning.h:32:2: error: #error This file requires compiler and library support for the ISO C++ 2011 standard. This support must be enabled with the -std=c++11 or -std=gnu++11 compiler options.
 #error This file requires compiler and library support \
  ^
In file included from /home/linux001/9521/vcpkg/buildtrees/ftgl/src/de901d4e74-1e9729eccc/src/FTGL/ftgl.h:137:0,
                 from /home/linux001/9521/vcpkg/buildtrees/ftgl/src/de901d4e74-1e9729eccc/src/FTBuffer.cpp:28:
/home/linux001/9521/vcpkg/buildtrees/ftgl/src/de901d4e74-1e9729eccc/src/FTGL/FTLibrary.h:137:14: error: ‘atomic’ in namespace ‘std’ does not name a template type
         std::atomic <int> LegacyOpenGLStateHandling;
              ^
/home/linux001/9521/vcpkg/buildtrees/ftgl/src/de901d4e74-1e9729eccc/src/FTGL/FTLibrary.h: In member function ‘bool FTLibrary::GetLegacyOpenGLStateSet() const’:
/home/linux001/9521/vcpkg/buildtrees/ftgl/src/de901d4e74-1e9729eccc/src/FTGL/FTLibrary.h:97:55: error: ‘LegacyOpenGLStateHandling’ was not declared in this scope
         bool GetLegacyOpenGLStateSet() const { return LegacyOpenGLStateHandling; }
```
It seems that this port still doesn't support Linux now.

--- a/ports/ftgl/CONTROL
+++ b/ports/ftgl/CONTROL
@@ -1,5 +1,6 @@
 Source: ftgl
 Version: 2.4.0-2
+Homepage: https://github.com/frankheckenbach/ftgl
 Description: FTGL is a free open source library to enable developers to use arbitrary fonts in their OpenGL (www.opengl.org) applications.
   Unlike other OpenGL font libraries FTGL uses standard font file formats so doesn't need a preprocessing step to convert the high quality font data into a lesser quality, proprietary format.
   FTGL uses the Freetype (www.freetype.org) font library to open and 'decode' the fonts. It then takes that output and stores it in a format most efficient for OpenGL rendering.

--- a/ports/ftgl/CONTROL
+++ b/ports/ftgl/CONTROL
@@ -1,5 +1,5 @@
 Source: ftgl
-Version: 2.4.0-1
+Version: 2.4.0-2
 Description: FTGL is a free open source library to enable developers to use arbitrary fonts in their OpenGL (www.opengl.org) applications.
   Unlike other OpenGL font libraries FTGL uses standard font file formats so doesn't need a preprocessing step to convert the high quality font data into a lesser quality, proprietary format.
   FTGL uses the Freetype (www.freetype.org) font library to open and 'decode' the fonts. It then takes that output and stores it in a format most efficient for OpenGL rendering.

--- a/ports/ftgl/portfile.cmake
+++ b/ports/ftgl/portfile.cmake
@@ -16,7 +16,13 @@ vcpkg_configure_cmake(
 )
 
 vcpkg_install_cmake()
-vcpkg_fixup_cmake_targets(CONFIG_PATH cmake)
+if (VCPKG_TARGET_IS_WINDOWS)
+        vcpkg_fixup_cmake_targets(CONFIG_PATH cmake)
+elseif (VCPKG_CMAKE_SYSTEM_NAME STREQUAL "Linux")
+        vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake)
+elseif (VCPKG_CMAKE_SYSTEM_NAME STREQUAL "Darwin")
+        vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake)
+endif()
 vcpkg_test_cmake(PACKAGE_NAME FTGL)
 
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)

--- a/ports/ftgl/portfile.cmake
+++ b/ports/ftgl/portfile.cmake
@@ -1,4 +1,3 @@
-include(vcpkg_common_functions)
 vcpkg_find_acquire_program(DOXYGEN)
 
 vcpkg_from_github(
@@ -18,14 +17,10 @@ vcpkg_configure_cmake(
 vcpkg_install_cmake()
 if (VCPKG_TARGET_IS_WINDOWS)
         vcpkg_fixup_cmake_targets(CONFIG_PATH cmake)
-elseif (VCPKG_CMAKE_SYSTEM_NAME STREQUAL "Linux")
-        vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake)
-elseif (VCPKG_CMAKE_SYSTEM_NAME STREQUAL "Darwin")
+else ()
         vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake)
 endif()
-vcpkg_test_cmake(PACKAGE_NAME FTGL)
 
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
 
-file(COPY ${SOURCE_PATH}/COPYING DESTINATION ${CURRENT_PACKAGES_DIR}/share/ftgl)
-file(RENAME ${CURRENT_PACKAGES_DIR}/share/ftgl/COPYING ${CURRENT_PACKAGES_DIR}/share/ftgl/copyright)
+file(INSTALL ${SOURCE_PATH}/COPYING DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)

--- a/scripts/ci.baseline.txt
+++ b/scripts/ci.baseline.txt
@@ -486,7 +486,6 @@ fribidi:arm-uwp=fail
 fribidi:x64-linux=fail
 fribidi:x64-osx=fail
 fribidi:x64-uwp=fail
-ftgl:x64-linux=fail
 ftgl:x64-osx=fail
 ftgl:x64-uwp=fail
 fuzzylite:arm-uwp=fail

